sdk/attestation/attestation/test/utils/utf8.browser.ts

```diff
@@ -1,32 +0,0 @@
-// Copyright (c) Microsoft Corporation.
```

Comment:
is the mapping for this file has been removed from package.json?

Reply:
I don't think it ever was in package.json. Checking.

Reply:
Fixing the name.